app.rb

```diff
@@ -1,9 +1,54 @@
 require 'sinatra/base'
+require './lib/peeps'
+require './lib/sequence'
+require './lib/database_connection'
+require './lib/filter'
+require './database_connection_setup'
+require 'pg'
 
 class Chitter < Sinatra::Base
   get '/test' do
     'Test page'
   end
 
+  get '/chitter' do
```

Comment:
There's a lot going on in this controller action.  Seems like you get all the peeps several times, which is confusing 🤔